pkg/agent/sysadvisor/plugin/qosaware/resource/memory/headroompolicy/policy_numa_aware.go

```diff
@@ -118,23 +117,11 @@ func (p *PolicyNUMAAware) Update() (err error) {
 		if err != nil {
 			return err
 		}
-		numaFree := data.Value
-
-		reclaimedCoresUsed := 0.
-		for _, container := range reclaimedCoresContainers {
-			data, err = p.metaServer.GetContainerNumaMetric(container.PodUID, container.ContainerName, strconv.Itoa(numaID), consts.MetricsMemTotalPerNumaContainer)
-			if err != nil {
-				general.ErrorS(err, "failed to get metric", "name", consts.MetricsMemTotalPerNumaContainer,
-					"podName", container.PodName, "ContainerName", container.ContainerName, "numaID", numaID)
-				return err
-			}
-			reclaimedCoresUsed += data.Value
-			general.InfoS("container memory", "podName", container.PodName, "containerName", container.ContainerName, "numaID", numaID, "memory used", general.FormatMemoryQuantity(data.Value))
-		}
-
-		reclaimableMemory += numaFree + reclaimedCoresUsed
+		reclaimableMemory += data.Value
```

Comment:
It's better to print logs like L137.

Reply:
we use reclaimed request, so memory reclaimable on each numa could not be calculated.

Reply:
numaFree need to be audited.

Reply:
> numaFree need to be audited.

numaFree is fine.